casanovo/casanovo.py

```
@@ -381,6 +401,27 @@ def configure(
     logger.info(f"Wrote {config_path}")
 
 
+def _warn_if_invalid(model: Optional[str], load_all_states: bool):
```

Ruff (0.15.6)

[warning] Missing return type annotation for private function `_warn_if_invalid`

Add return type annotation: `None`

(ANN202)